docs/proposals/connector-source-simplified-state-updates.md

```diff
@@ -0,0 +1,147 @@
+# Simplified stat eupdate sequences for the connector source protocol with sources on multiple workers and source migration
```

Comment:
I'm not sure we want this document.  Did you intend to leave it in here?

Reply:
@nisanharamati do we want this here?

Reply:
I think we should merge it into the main protocol doc